src/people/widgetViews/summaries/wantedSummaries/__tests__/CodingBounty.spec.tsx

```diff
@@ -163,4 +171,158 @@ describe('MobileView component', () => {
+    });
+
+    render(<MobileView {...props} />);
+
+    fireEvent.click(screen.getByText('Not Assigned'));
+
+    await waitFor(() => {
+      expect(screen.getByText('Assign Developer')).toBeInTheDocument();
+      expect(screen.getByPlaceholderText('Type to search ...')).toBeInTheDocument();
+      expect(screen.getByText('Skills')).toBeInTheDocument();
+      expect(document.querySelectorAll('.PeopleList .People')).toHaveLength(5);
+    });
+  });
+
+  it('Test filter peopleList by "type to search" or "skills"', async () => {
+    const props: CodingBountiesProps = {
+      ...defaultProps,
+      assigneeValue: true,
+      creatorStep: 0,
+      peopleList: [
+        { id: 1, owner_alias: '111' },
+        { id: 2, owner_alias: '222' },
+        { id: 3, owner_alias: '333' },
+        { id: 4, owner_alias: '444' },
+        { id: 5, owner_alias: '555' }
+      ] as any
+    };
+
+    uiStore.setMeInfo({
+      ...user,
+      owner_alias: props.person.owner_alias
+    });
+
+    const mockPeopleList: any = [
+      {
+        id: 1,
+        owner_alias: 'TEST_NAME_1',
+        extras: {
+          coding_languages: [
+            { value: 'R', label: 'R' },
+            { value: 'C++', label: 'C++' }
+          ]
+        }
+      },
+      {
+        id: 2,
+        owner_alias: 'TEST_NAME_2',
+        extras: { coding_languages: [{ value: 'C', label: 'C' }] }
+      },
+      { id: 3, owner_alias: 'TEST_NAME_3', extras: { coding_languages: [] } }
+    ];
+    const mockSearch = jest
+      .spyOn(mainStore, 'getPeopleByNameAliasPubkey')
+      .mockResolvedValue(mockPeopleList);
+
+    render(<MobileView {...props} />);
+
+    fireEvent.click(screen.getByText('Not Assigned'));
+
+    // filter by "type to search"
+    await waitFor(async () => {
+      expect(screen.getByText('Assign Developer')).toBeInTheDocument();
+      fireEvent.click(screen.getByPlaceholderText('Type to search ...'));
+      await userEvent.type(screen.getByPlaceholderText('Type to search ...'), 'TEST_NAME');
+      expect(document.querySelectorAll('.PeopleList .People')).toHaveLength(mockPeopleList.length);
+
+      mockPeopleList.forEach((person: any) => {
+        expect(screen.getByText(person.owner_alias)).toBeInTheDocument();
+      });
+    });
+
+    // filter by "skills"
+    await waitFor(() => {
+      fireEvent.click(screen.getByText('Skills'));
+      fireEvent.click(screen.getByText('R'));
+
+      fireEvent.keyDown(document, { key: 'Escape', keyCode: 27 });
+    });
+
+    expect(document.querySelectorAll('.PeopleList .People')).toHaveLength(1);
+    expect(screen.getByText('TEST_NAME_1')).toBeInTheDocument();
+    expect(screen.queryByText('TEST_NAME_2')).not.toBeInTheDocument();
+    expect(screen.queryByText('TEST_NAME_3')).not.toBeInTheDocument();
+
+    mockSearch.mockRestore();
+  });
+
+  it('Test that on clicking on "invite" on a hunter, the pop up should clear and the hunter should be assigned to the bounty', async () => {
```

Comment:
@arimusdev Can you change the description of this test from "Invite" to  "Assign"